search/api_test.py

```diff
@@ -476,3 +477,27 @@ def test_populate_query_memberships(self, source_type, is_member, query_matches,
             membership = PercolateQueryMembership.objects.get(user=profile.user, query=query)
             assert membership.is_member is query_matches
             assert membership.needs_update is True
+
+    @ddt.data(*product(
+        [True, False],
+        [True, False]
+    ))
```

Comment:
Instead of using `*product` here, can you just put in all the different combinations and add parameters for the expected `percolate` call count and the expected number of `PercolateQueryMembership` objects? If I'm not mistaken that would be just 4 sets of parameters and would be more readable